exploration:
  dim_entropy_map:      3
  use_rdcd_map_entropy: false
  max_speed:            0.25
  height:               2.0
  alpha_xy_factor:      3
  alpha_z_factor:       3
  free_nbrs_x:          3
  free_nbrs_y:          3
  free_nbrs_z:          1
  min_info_gain_thresh: 0.02
  num_cells_consider:   3
  poly_order:           2
  points_per_segment:   8
  traj_time_step:       0.01
  timers_on:            true
  yaw_rotation_time:    20.0
  gazebo_load_time:     3.0
  gazebo_topic:         /gazebo/set_model_state
  min_max_thresh:       0.11
  prob_H_map_thresh:    0.01
  num_rays_per_cand_z:  16
  num_rays_per_cand_y:  9
  acceptable_coll_prob: 0.05
  candidate_separation: 0.5
  recompute_radius:     5.0
  FOV_angle_z:          1.6
  FOV_angle_y:          0.78539816339
  deg_cam_pitch:        5.0
  bump_dist:            10.0
  bump_val_at_dist:     0.5
  min_bump_val:         0.1
  discount_radius:      3.0
  max_computation_time: 0.125
  max_angular_vel:      0.6
  limit_cost_map:       true
  num_cells_cost_map:   20000
  distance_bump_BMax:   1.0
  distance_bump_BFar:   0.1
  distance_bump_beta:   0.1
  full_map_pass_time:   10.0
  edge_cells_visited:   true
  # Topics get Namespaced in Code (viz topics too)
  camera_path_topic:    CPLS_camera
  position_path_topic:  CPLS_position
  velocity_path_topic:  CPLS_velocity
  accelera_path_topic:  CPLS_acceleration
  traj_info_topic:      explore_traj_info
  des_pose_topic:       desired_pose
  des_cam_topic:        desired_cam_pose
  des_twist_topic:      desired_twist
  des_accel_topic:      desired_accel
  visualization:
    ogm_topic:              ogm_2D_coll
    pub_paths:              false
    viz_cost_map:           true
    cost_map_topic:         cost_map
    viz_cand_entropies:     false
    entropy_marker_topic:   info_gains_candidates
    viz_cand_entr_dists:    true
    dist_info_marker_topic: info_gains_distances_candidates
    arrow_scale:            0.2
    djk_topic:              DJK
    non_optimal_opacity:    0.6
    arrow_length:           0.55
    arrow_shaft_diameter:   0.15
    arrow_head_diameter:    0.25
    arrow_head_length:      0.20
